saleor/order/notifications.py

```diff
@@ -243,7 +244,8 @@ def get_default_order_payload(order: "Order", redirect_url: str = ""):
     order_payload = model_to_dict(order, fields=ORDER_MODEL_FIELDS)
     order_payload.update(
         {
-            "number": order.id,
+            "id": to_global_id_or_none(order),
+            "number": to_global_id_or_none(order),
```

Comment:
This is incorrect. `number` field should return a number of a given order.

Reply:
This is the same thing actually:
```
@staticmethod
    def resolve_number(root: models.Order, _info):
        return str(root.pk)
```

Reply:
This is the same thing actually:
```
@staticmethod
    def resolve_number(root: models.Order, _info):
        return str(root.pk)
```
in `saleor.graphql.order.types.Order`

Reply:
It works because we didn't merge conversion from int to UUID yet.

For your case, it should be:
```
id: base64("Order", order.id)
number: order.id (as it is still int value)

Currently it doesn't make sense as both fields return the same value.

Reply:
I merged the ID migration today so this PR can be updated and we can see if it works as expected now.